perf/ring.go

```diff
@@ -22,12 +22,19 @@ type perfEventRing struct {
 	*ringReader
 }
 
-func newPerfEventRing(cpu, perCPUBuffer, watermark int) (*perfEventRing, error) {
+
+// In C, struct perf_event_attr has a write_backward field which is a bit in a
+// 64-length bitfield.
+// In Golang, there is a Bits field which 64 bits long.
+// From C, we can deduce write_backward is the is the 27th bit.
+const writeBackwardBit = 1 << 27
```

Comment:
Could use:
```
const perfBitWriteBackward = unix.CBitFieldMaskBit27
```
to keep the naming style from the unix package.

See:
https://cs.opensource.google/go/x/sys/+/refs/tags/v0.1.0:unix/ztypes_linux.go;l=1009
https://cs.opensource.google/go/x/sys/+/refs/tags/v0.1.0:unix/ztypes_linux_amd64.go;drc=95e765b1cc43ac521bd4fd501e00774e34401449;l=388

That would automatically benefit from supporting other architectures with other endianness:
```
unix/ztypes_linux_ppc64.go:326: CBitFieldMaskBit26 = 0x2000000000
unix/ztypes_linux_amd64.go:337: CBitFieldMaskBit26 = 0x4000000
```